src/global/helpers/timezone-helper.ts

```diff
@@ -0,0 +1,17 @@
+import { isValid, parseISO } from 'date-fns';
```

Comment:
what generally let me think about this approach is, eg you have a timestamp ""2022-08-28T23:00:00+01:00"" and "2022-08-29T00:00:00+02:00" which is the exact same moment, leads to two different dates.  Do we maybe need to convert to UTC first? or let the user chose in which timezone he wants the date converted first?